lib/screens/language_domain/language_game_three.dart

```diff
@@ -0,0 +1,267 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:http/http.dart' as http;
+import 'package:flutter/material.dart';
+import 'dart:async';
+import 'dart:math';
+import 'enum.dart';
+
+class LanguageGameThree extends StatefulWidget {
+  const LanguageGameThree({super.key});
+
+  @override
+  State<LanguageGameThree> createState() => _LanguageGameThreeState();
+}
+
+class _LanguageGameThreeState extends State<LanguageGameThree> {
+  final String baseUrl = Platform.isAndroid
+      ? 'http://192.168.1.2:8080/api/language'
+      : 'http://localhost:8080/api/language';
+  final Uri dictionaryUrl =
+      Uri.parse('https://vietnamese-wordlist.duyet.net/Viet74K.txt');
```

Comment:
Make a constant variable in a file inside constants folder

Reply:
Done